test/integration/discord.test.js

```diff
@@ -75,4 +78,74 @@ describe("test discord actions", function () {
         });
     });
   });
+
+  describe("test discord actions for nickname for verified user", function () {
+    beforeEach(async function () {
+      fetchStub = sinon.stub(global, "fetch");
+      superUser = { ...userData[4], discordId: "123456789" };
+      userId = await addUser();
+      superUserId = await addUser(superUser);
+      superUserAuthToken = authService.generateAuthToken({ userId: superUserId });
+    });
+
+    afterEach(async function () {
+      await cleanDb();
+      sinon.restore();
+    });
+
+    it("returns 200 for updating nickname post method", function (done) {
+      fetchStub.returns(
+        Promise.resolve({
+          status: 200,
+          json: () => Promise.resolve({}),
+        })
+      );
+      chai
+        .request(app)
+        .post("/discord-actions/nickname")
+        .set("Cookie", `${cookieName}=${superUserAuthToken}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body.message).to.equal("nickname has been changed");
```

Comment:
would it be a good idea to even send the old name and updated name in the response as It will allow clients to validate that the expected change was made  correctly.
Something of the sort 
```json
{
  "status": 200,  
  "message": "nickname has been changed",               
  "data": {
     "previous": "Khushi",
     "updated": "Amazing Khushi"
   }
}
```

Reply:
as you see in the api , There are no previous and updated username. these will be updated from mysite so there is no need to add this.